docs/_static/basic.css

```diff
@@ -4,7 +4,7 @@
  *
  * Sphinx stylesheet -- basic theme.
  *
- * :copyright: Copyright 2007-2019 by the Sphinx team, see AUTHORS.
+ * :copyright: Copyright 2007-2016 by the Sphinx team, see AUTHORS.
```

Comment:
@minhtue1996 we are still using sphinx==1.3.6 to generate the doc

Reply:
It's a bit weird that these changes happen then. Though it might not be worth to look into it.
Also I forgot to ask, but how can I view the doc change in browser? Usually before, I would run `make html; make upload_html` inside the `docs/` folder of the `legup` repo, which will upload the html docs to a LegupComputing. The reviewer can then view the docs on their browser using a link.

For more information, see [docs/Makefile ](https://bitbucket.microchip.com/projects/HLS/repos/legup/browse/docs/Makefile)

Example of the user guides on LegupComputing: http://10.245.245.140/user_guides/